docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Get started with Azure Blob storage using F#
+description: Store unstructured data in the cloud with Azure Blob storage.
+keywords: visual f#, f#, functional programming, .NET, .NET Core, Azure
+author: sylvanc
+manager: jbronsk
+ms.date: 09/20/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: c5b74a4f-dcd1-4849-930c-904b6c8a04e1
+---
+
+# Get started with Azure Blob storage using F# #
+
+Azure Blob storage is a service that stores unstructured data in the cloud as objects/blobs. Blob storage can store any type of text or binary data, such as a document, media file, or application installer. Blob storage is also referred to as object storage.
+
+### About this tutorial
```

Comment:
I think you don't need this heading and the next one. and then "Create an Azure storage account" should perhaps be called "Prerequisites" in a H2 heading instead of H3.